### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by Khaaz** [Khaaz/Cross-Server-Bot#20]
# v3.1.0
- fix enhanced mention (use discord `allowedMentions`)
- support true edit, and use webhook delete
- support for discord reply
- option so all message deletion gets waterfalled to all servers and not only the main message deletion (current behaviour)
- support transfering embeds through webhooks

Any PR that helps toward this goal will be accepted.

# v4.0.0 and later

For the future of this bot we could expect:
- auto creation and management of webhook creation deletion (optional)
- multiple channel network on the same bot

**Comment by X-Niter** [comment]
In reference to multiple channel networks, is this referring to being able to let's say sync 4 different channels together but is pairs of 2.
So channels 1 & 2 will sync and channel 3 & 4 will sync, meaning they are not all synced together.  ??

**Comment by Khaaz** [comment]
Yes exactly. What I call a network is a set of channels linkeds together in sync.
Right now, you have to host many instance of this bot in order to have several network of channels. It's fine,you can even use the same token. However it's not optimal so doing that within the same bot is ultimately better.

**Comment by X-Niter** [comment]
Ah, Understood

**Comment by joeydoesthings** [comment]
Any approximate release date?

**Comment by bigry8058** [comment]
Khazz Ive been hammering away at the message deletion and I've gotten it to where it works about 70% of the time but it may randomly crash, my solution was a simple script to restart on crash. I will share my changes when I get it working 100% of the time. Replies are a different story.

**Comment by Re4LSteaL** [comment]
Hello, any update on this?
